src/util/highlight.js

```diff
@@ -0,0 +1,113 @@
+// copied from React InstantSearch
+const HIGHLIGHT_TAGS = {
+  highlightPreTag: `<ais-highlight-0000000000>`,
+  highlightPostTag: `</ais-highlight-0000000000>`,
+};
+
+const getPropertyByPath = (object, path) =>
```

Comment:
We have this helper already in lib/object or something like that

Reply:
https://github.com/algolia/vue-instantsearch/pull/783/commits/caba208088709f649e81ab4a6c4dd9a06e9e7be5